src/Workleap.Authentication.ClientCredentialsGrant.Tests/XunitLoggerProvider.cs

```diff
@@ -32,7 +25,7 @@ public ILogger CreateLogger(string categoryName)
     public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
     {
         var message = formatter(state, exception);
-        this._output.WriteLine("[{0:HH:mm:ss:ffff} {1}] {2}", DateTime.Now, LogLevelStrings[logLevel], message);
+        testOutput.WriteLine("[{0:HH:mm:ss:ffff} {1} {2}] {3}", DateTime.Now, appName, LogLevelStrings[logLevel], message);
```

Comment:
Helps distinguish the originating app of each log entry:

![image](https://github.com/user-attachments/assets/ddee5045-924d-44ce-8d1a-b02e7968591f)